src/schemaGenerator.ts

```diff
@@ -7,7 +7,7 @@
 // a bunch of utility functions into a separate utitlities folder, one file per function.
 
 import { DocumentNode, parse, print, Kind, DefinitionNode } from 'graphql';
-import { uniq } from 'lodash';
+import uniq = require('lodash.uniq');
```

Comment:
@DxCx here is the [same usage](https://github.com/colevoss/apollo-client/blob/master/src/core/QueryManager.ts#L7-L8) in `react-apollo`, it's also used this way in another apollo packages. Since I don't know ts, I can only imagine why only this weird syntax works 😄